src/types/index.ts

```diff
@@ -103,13 +103,13 @@ export type AnonymizedResultColumn = {
 
 export type AnonymizedColumnType = ColumnType | 'aggregate';
 
-export type AnonymizedResultRow = { lowCount: boolean; values: AnonymizedValue[] };
-
-export type AnonymizedValue = Value | AnonymizedAggregate;
-
-export type AnonymizedAggregate = {
-  realValue: number;
-  anonValue: number | null;
+// Represents row data returned from the preview query request to the
+// `OpenDiffix.Service`
+export type AnonymizedResultRow = {
+  lowCount: boolean;
+  count: number;
+  diffixCount: number | null;
+  bucketValues: RowData;
 };
```

Comment:
The star bucket could be returned from `OpenDiffix.Service` to match other rows so that we don't have to build it manually. Then we don't have to care about matching column types and values.